apps/rush-lib/src/cli/actions/PublishAction.ts

```diff
@@ -508,4 +507,18 @@ export class PublishAction extends BaseRushAction {
       }
     }
   }
+
+  private _getRegistryAndUpdateEnv(env: { [key: string]: string | undefined }): string {
+    let registry: string = '//registry.npmjs.org/';
+    if (this._registryUrl.value) {
+      const registryUrl: string = this._registryUrl.value;
+      env['npm_config_registry'] = registryUrl; // tslint:disable-line:no-string-literal
```

Comment:
can you not do `env.npm_config_registry`?

```suggestion
      env.npm_config_registry = registryUrl;
```

Reply:
oh wait i see this is just a copy & paste

Reply:
Maybe? I was just following the existing style since this code was moved from elsewhere in the file.

Reply:
Eventually we should make the environment more structured, but for now let's leave this as-is.